hoomd/BoxDim.h

```diff
@@ -609,24 +609,24 @@ struct
     //! Serialization method
     template<class Archive> void serialize(Archive& ar, const unsigned int version)
         {
-        ar& m_lo.x;
-        ar& m_lo.y;
-        ar& m_lo.z;
-        ar& m_hi.x;
-        ar& m_hi.y;
-        ar& m_hi.z;
-        ar& m_L.x;
-        ar& m_L.y;
-        ar& m_L.z;
-        ar& m_Linv.x;
-        ar& m_Linv.y;
-        ar& m_Linv.z;
-        ar& m_xy;
-        ar& m_xz;
-        ar& m_yz;
-        ar& m_periodic.x;
-        ar& m_periodic.y;
-        ar& m_periodic.z;
+        ar & m_lo.x;
```

Comment:
I'm personally not a fan of the space being added here when defining a reference variable. It also seems to apply this rule inconsistently, as the function argument doesn't have a space between the variable type and the ampersand.